src/pfe/portal/modules/User.js

```diff
@@ -659,6 +670,288 @@ module.exports = class User {
     return retval;
   }
 
+  /**
+   * Function to setup the Docker config file and Kube secret
+   */
+  async setupRegistrySecret(username, password, url) {
+    log.info("Setting up the Docker Registry secret for url: " + url + " and username: " + username);
```

Comment:
Does it matter? Because username is public to everyone who has access to CW. But if we are worried about it being retinaed in the logs, we can remove it